src/Util.js

```diff
@@ -142,6 +142,10 @@ function loadStyleFromUrl (styleUrl, options, callback) {
   request(styleUrl, params, callback);
 }
 
+function isSameTLD (url1, url2) {
+  return (new URL(url1)).hostname === (new URL(url2)).hostname;
```

Comment:
I do have some mild concerns about this because `new URL` will throw errors on relative paths. In general I think this is ok because we check if they are relative in `formatStyle`. Ideally I would love to use `new URL()` to resolve the paths

Reply:
> I do have some mild concerns about this because `new URL` will throw errors on relative paths. In general I think this is ok because we check if they are relative in `formatStyle`. 

I agree - I think it's not ideal but ok in this case since we're checking for relative in the parent function.

> Ideally I would love to use `new URL()` to resolve the paths
I'm not following here - are you suggesting a code change for this PR or just a general future idea?

Reply:
General future idea.